drivers/telescope/lx200telescope.cpp

```diff
@@ -1380,28 +1380,54 @@ bool LX200Telescope::sendScopeLocation()
         IDSetNumber(&LocationNP, nullptr);
         return true;
     }
-
-    if (getSiteLatitude(PortFD, &lat_dd, &lat_mm, &lat_ssf) < 0)
-    {
-        LOG_WARN("Failed to get site latitude from device.");
-        return false;
-    }
-    else
-    {
-        snprintf(lat_sexagesimal, MAXINDIFORMAT, "%02d:%02d:%02.1lf", lat_dd, lat_mm, lat_ssf);
-        f_scansexa(lat_sexagesimal, &(LocationNP.np[LOCATION_LATITUDE].value));
+    
+    
+    if (genericCapability & LX200_HAS_ALTERNATE_LOCATION_CMD) {
+        if (getSiteLatitudeAlt(PortFD, &lat_dd, &lat_mm, &lat_ssf, ":GtH#") < 0)
+        {
+            LOG_WARN("Failed to get site latitude from device.");
+            return false;
```

Comment:
I think instead of adding LX200_HAS_ALTERNATE_LOCATION_CMD, wouldn't it be easier if sendScopeLocation/Time are virtual and can be overrided by the child class?

This way you don't need to add LX200_HAS_ALTERNATE_LOCATION_CMD and it would make the base class simpler and less prone to regressions.